### PR TITLE
clean: remove unused code and reuse minimark

### LINKED ISSUE #3037: ⚰️: Remove dead/unused code from repo
We have a full implementation of RMRK Classic in the [minimark](https://github.com/vikiival/minimark)

Your task is:
- [ ] unused functions/classes from the repository
- [ ] remove duplicated types (there is already a ton of them in minimark)
- [ ] if you find some functionality that is tied to RMRK and is used here please [open an issue in minimark](https://github.com/vikiival/minimark/issues/new)

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### What's new?

- [x] PR closes #3037 

- remove unused functions/classes from the repository
- remove duplicated types (there is already a ton of them in minimark)
-  if you find some functionality that is tied to RMRK #ref: https://github.com/vikiival/minimark/issues/12

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [ ] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Review by prachi00 — APPROVED** [review]

**Comment by prachi00** [comment]
pay 200 usd

**Comment by yangwao** [comment]
pay 200 usd 

@Jarsen136 let's resolve conflicts :)

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $200 @ 83.06 USD/KSM ~ 2.408 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0xe7de6419f9ab6e7742eec2b9914e892e71c2212ced393381a526020b5a69c2f9](https://kusama.subscan.io/extrinsic/0xe7de6419f9ab6e7742eec2b9914e892e71c2212ced393381a526020b5a69c2f9)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by Jarsen136** [comment]
Conflicts have been resolved.

File: components/rmrk/service/NftUtils.ts
Comment (vikiival) [inline components/rmrk/service/NftUtils.ts:225]: Do we need this?

File: components/rmrk/utils.ts
Comment (vikiival) [inline components/rmrk/utils.ts:220]: is this used somewhere?

File: components/rmrk/utils.ts
Comment (vikiival) [inline components/rmrk/utils.ts:243]: is this used somewhere?

File: utils/ipfs.ts
Comment (vikiival) [inline utils/ipfs.ts:36]: 🤔 this is not part of minimark afaik

**Review by vikiival — CHANGES_REQUESTED** [review]
smol stuff

File: utils/ipfs.ts
Comment (Jarsen136) [reply utils/ipfs.ts:36]: Yes, but it also has never been used/called.

Comment (Jarsen136) [reply utils/ipfs.ts:36]: So I remove it.

File: components/rmrk/utils.ts
Comment (Jarsen136) [reply components/rmrk/utils.ts]: removed

File: components/rmrk/service/NftUtils.ts
Comment (Jarsen136) [reply components/rmrk/service/NftUtils.ts]: removed

**Comment by vikiival** [comment]
You have some conflicts sir

**Comment by prachi00** [comment]
@vikiival can you also approve once ? Unable to merge this one.

**Review by vikiival — APPROVED** [review]

**Pull request merged**
